common/patches/rof2.cpp

```diff
@@ -2746,6 +2747,18 @@ namespace RoF2
 			memcpy(&outlaa->raid, &inlaa->raid, sizeof(RaidLeadershipAA_Struct));
 			dest->FastQueuePacket(&outapp);
 		}
+		else if (raid_gen->action == raidSetNote)
+		{
+			auto in_note = (RaidGeneral_Struct*)__emu_buffer;
+			auto outapp = new EQApplicationPacket(OP_RaidUpdate, sizeof(RaidGeneral_Struct));
+			auto note = (RaidGeneral_Struct*)outapp->pBuffer;
+			note->action = raidSetNote;
+			strn0cpy(note->leader_name, in_note->leader_name, 64);
+			strn0cpy(note->player_name, in_note->player_name, 64);
+			strn0cpy(note->note, in_note->note, 64);
```

Comment:
`sizeof(note->leader_name)` instead of `64`.
`sizeof(note->player_name)` instead of `64`.
`sizeof(note->note)` instead of `64`.

Reply:
Updated.